topics/variant-analysis/tutorials/baculovirus-isolate-variation/tutorial.md

```diff
@@ -0,0 +1,851 @@
+>
+>    > <comment-title> What bcftools call does... </comment-title>
+>    > bcftools call performs the actual variant detection. It removes non variant and keeps variant sites only.
+>    {: .comment}
+>
+{: .hands_on}
+
+What you get is a file in Variant Call Format (VCF), which can be difficult to understand at first glance. It is important to note that the file begins with a large header, lines that begin with two hashtags (##), which contain details of the analysis and the data set. After the header, there is a table with a line that contains the column names. This line begins with just one hashtag (#). After that, the results are displayed in a tab-separated format. In our VCF example, some columns have the following abbreviations and meaning (I just select a few that are important to us in this tutorial):
+
+| Fixed fields  | Description  |
+|---:|:---|
+|**CHROM**  |Chromosome (name of the reference, in our case CpGV-M)  |
+|**POS**  |Position (position in the reference genome)   |
+|**REF**  |Nucleotide of the Referance at the corresponding position (POS)   |
+|**ALT**   |Alternative nucleotides detected at this position in the sequencing data.   |
+|**FORMAT**   |Describes the content of the *sample column* reported in the VCF file. In our case GT:PL:DP:DPR.           |
+|**SRR31589146**   |1st sample column: The results for the sequence data SRR31589146 in the format specified in FORMAT.   |
+|**SRR31589147**   |2nd sample column: The results for the sequence data SRR31589147 in the format specified in FORMAT.   |
+|**SRR31589148**   |3rd sample column: The results for the sequence data SRR31589148 in the format specified in FORMAT.   |
+|**SRR31679023**   |4th sample column: The results for the sequence data SRR31679023 in the format specified in FORMAT.   |
+
+
+To understand how the data is stored, we have to look at FORMAT in detail. This is where two values are of great importance: DP and DPR.
+
+|Genotype&nbsp;field|Description   |
+|---:|:---|
+|**DP**   |Read depth (number of nucleotides) at this position for this sample.   |
+|**DPR**   |The read depth for each allele. Here, the first value corresponds to the reference nucleotide (REF), the second to the first possible allele (ALT1), the second to the second possible allele (ALT2) and the third value to the last possible allele (ALT3).
+
+> <comment-title>DPR functional, but deprecated</comment-title>
+> If you start **bcftools mpileup** with the *Optional tags to output* `DPR (Number of high quality bases for each observed allele)` option, you will see a warning in the information panel of the VCF file: `[warning] tag DPR functional, but deprecated. Please switch to AD in future`. At the time of writing this tutorial, I was using DPR. It works the same way with *Optional tags to output* option `AD` (instead of DPR), but you will need to adjust something later in the workflow. The tutorial will be switched to AD in the near future. For now, we stick with DPR.
```

Comment:
```suggestion
> If you start **bcftools mpileup** with the *Optional tags to output* `DPR (Number of high-quality bases for each observed allele)` option, you will see a warning in the information panel of the VCF file: `[warning] tag DPR functional but deprecated. Please switch to AD in future. At the time of writing this tutorial, I was using DPR. It works the same way with *Optional tags to output* option `AD` (instead of DPR), but you will need to adjust something later in the workflow. The tutorial will be switched to AD in the near future. For now, we stick with DPR.
```